src/web/modules/webhooks/overview.njk

```diff
@@ -1,6 +1,6 @@
-x
 {% from "common/json.njk" import json %}
 {% extends "layout/layout.njk" %}
+{% from "./status.macro.njk" import agreementStatusTag %}
```

Comment:
should this macro be called `agreementStatusTag` if it's also used for other status tags?